assets/shaders/irradiance_volume_voxel_visualization.wgsl

```diff
@@ -0,0 +1,17 @@
+#import bevy_pbr::forward_io::VertexOutput
+#import bevy_pbr::irradiance_volume
+#import bevy_pbr::mesh_view_bindings
+
+@fragment
+fn fragment(mesh: VertexOutput) -> @location(0) vec4<f32> {
+    // Using `irradiance_volumes[0]` is a little dodgy, but we know we only have
+    // one irradiance volume in this scene, so it's fine.
+    //
+    // `irradiance_volume_light()` multiplies by intensity, so cancel it out.
+    // If we take intensity into account, these spheres will be way too bright.
+    let rgb = irradiance_volume::irradiance_volume_light(
+        mesh.world_position.xyz,
```

Comment:
this will not sample the exact center of the voxel will it? it might be worthwhile to do some coordinate snapping math to ensure the exact color stored in the voxel is reproduced

Reply:
OK, I do that now.